db/migrate/20220811135344_add_category_to_books.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryToBooks < ActiveRecord::Migration[6.1]
+  def change
+    add_belongs_to :books, :category
```

Comment:
move to create_books migration and add index and foreign key

Reply:
fixed